.gitignore

```diff
@@ -139,3 +139,6 @@ dist
 # Backup directories
 ./igbo_api_db
 ./aws_pronunciations
+
+# misc
```

Comment:
Why add a misc section? We're these files getting added in your commits?

Reply:
Yes the file is present in the project. It is usually ignored.
<img width="983" alt="Screen Shot 2021-07-06 at 15 40 48" src="https://user-images.githubusercontent.com/33348046/124610937-3eba4e80-de71-11eb-841b-3590dc43f099.png">